src/test/kotlin/org/jetbrains/bio/span/Span2CLALongTest.kt

```diff
@@ -0,0 +1,200 @@
+package org.jetbrains.bio.span
+
+import org.jetbrains.bio.Configuration
+import org.jetbrains.bio.Tests
+import org.jetbrains.bio.experiments.fit.SpanModel
+import org.jetbrains.bio.genome.Genome
+import org.jetbrains.bio.io.BedFormat
+import org.jetbrains.bio.query.reduceIds
+import org.jetbrains.bio.query.stemGz
+import org.jetbrains.bio.statistics.distribution.Sampling
+import org.jetbrains.bio.util.*
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class Span2CLALongTest {
```

Comment:
Why do we need this test when `SpanCLALongTest` deals with `.span2` models already?

Reply:
It deals with them by checking that they're not compatible with `--type nbhmm` option. `Span2CLALongTest` contains the actual Span2 tests, like program output and files created.

Reply:
In other words, `SpanCLA` is enormous as it is, and I thought moving at least part of tests to another file would improve the readability. If it's not the case, I'll easily merge them back.